yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -296,8 +308,7 @@ export class Sequencer extends (EventEmitter as new () => TypedEventEmitter<Sequ
     const chainTipArchive = syncedTo.archive;
     const newBlockNumber = syncedTo.blockNumber + 1;
 
-    const { slot, ts, now } = this.publisher.epochCache.getEpochAndSlotInNextL1Slot();
-    this.metrics.observeSlotChange(slot, this.publisher.getSenderAddress().toString());
```

Comment:
@alexghr I wasn't really sure what to do here. Is it ok to call this with different publisher addresses.